### ISSUE TITLE
terminate called after throwing an instance of 'std::bad_cast'

### ISSUE THREAD

**Issue by Dracnea** [iron-fish/ironfish-node-app#168]
### What happened?

System Information: Ubuntu 20 VM within Proxmox
Using AppImage version of IronFish Wallet
Issue occurs after having imported wallet using passphrase and selecting "quick and centralized" block updating.
Every time I start application is processes another 1-3 block sequences in increments of 100.

### Version

v2.0.2

### Debug output

```shell
./Iron-Fish-Node-App-2.0.2.AppImage debug
01:45:32.409 › Checking for update
[3225:0201/014532.644435:ERROR:viz_main_impl.cc(196)] Exiting GPU process due to errors during initialization
01:45:33.224 › Initializing Iron Fish SDK...
01:45:33.391 › Update for version 2.0.2 is not available (latest version: 2.0.2, downgrade is disallowed).
01:45:33.392 › checkForUpdatesAndNotify called, downloadPromise is null
01:45:33.570 › Starting Iron Fish Node...
01:45:33.630 › [node] WebSocket server started at :::9033
01:45:33.715 › Started Iron Fish Node.
01:45:34.019 › [node] Connected to the Iron Fish network
01:45:34.981 › [node] Added block seq: 1300, hash: 00000...8ed3c
terminate called after throwing an instance of 'std::bad_cast'
  what():  std::bad_cast
Aborted (core dumped)
```

**Comment by dguenther** [comment]
Ah, we don't currently support a Linux release due to the above issue, so I should remove those AppImage files from the releases.

The bug mentioned above is due to some issue with node-datachannel when compiled in an Electron package -- I think we'll have to rebuild the package targeting Linux before we generate the Linux Electron build.

**Comment by GamerWierdo100** [comment]
You should really remove the appimage download, it lead me down a big rabbit hole before finding this

**Comment by dguenther** [comment]
I found a potential fix: https://github.com/iron-fish/ironfish-node-app/pull/195 I'll update this issue once it's reviewed and a new release is out.

If that PR doesn't fix it, I'll add a note to the known issues in the README and the release notes, but I'd lean toward not removing the AppImage downloads for now because this crash doesn't seem to happen on Ubuntu, and I'd rather not drop support for those users.

**Comment by dguenther** [comment]
This is now fixed as of v3.0.4 on Fedora 39 for me, so I'll close this issue. If anyone is still running into the crash, please reopen and share your Linux distribution info.
